tests/test_sample_slices.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from monai.utils import sample_slices
+from tests.utils import TEST_NDARRAYS, assert_allclose
+
+TEST_CASE_1 = [torch.tensor([[[0, 2], [1, 0]]]), 1, (1,), (1, 1, 2), torch.tensor([[[1, 0]]])]
+
+TEST_CASE_2 = [torch.tensor([[[0, 2], [1, 0], [4, 5]]]), 1, (0, 2), (1, 2, 2), torch.tensor([[[0, 2], [4, 5]]])]
+
+
+class TestSampleSlices(unittest.TestCase):
+    @parameterized.expand([TEST_CASE_1, TEST_CASE_2])
+    def test_shape(self, input_data, dim, vals, expected_shape, expected_result):
+        for p in TEST_NDARRAYS:
+            result = sample_slices(p(input_data), dim, *vals)
+
+            self.assertEqual(result.shape, expected_shape)
```

Comment:
shape check is redundent here if the `expected_result` are compared with assert_allclose

Reply:
> > Hi @wyli ,
> > BTW, I think maybe something wrong updated in the CI configs?
> > ```
> > Error while finding module specification for 'pip' (AttributeError: module '__main__' has no attribute '__file__')
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Thanks.
> 
> looks like it's from setuptools and the new version is yanked https://pypi.org/project/setuptools/60.3.0/

Hi @wyli ,

I see your PR https://github.com/Project-MONAI/MONAI/pull/3597 passed the CI tests, so you have updated something to fix the CI software issue?

Thanks.

Reply:
Sure, removed in the latest commit.

Thanks.